developer-docs-site/docs/guides/guide-for-system-integrators.md

```diff
@@ -1,43 +1,44 @@
-
 :::tip 
-This is an ongoing documentation. We will add more sections on a daily basis.
+This is documentation is currently under construction with more being added on a regular basis.
 :::
 
-## Prerequisites
+# System Integrators' Guide
 
-This guide assumes that you are familiar with the blockchains in general,  and in specific the command-line interfaces and other concepts discussed throughout this Aptos developer documentation site. 
+If you provide blockchain services to your customers and wish to add the Aptos blockchain to your platform, then this guide is for you. This system integrators guide will walk you through all you need to integrate the Aptos blockchain into your platform. This guide assumes that you are familiar with the blockchains.
 
 ## Overview
 
-As a system integrator you will perform some or all of these below tasks on behalf of your users: 
-
+This guide will overview the following core concepts for integrating with Aptos:
 - Create an account on the blockchain.
 - Exchange account identifiers with another entity on the blockchain, for example, to perform swaps.
 - Create a transaction.
 - Obtain a gas estimate and validate the transaction for correctness.
-- Ask the user to verify the intent and cost of the transaction.
 - Submit the transaction to the blockchain.
 - Wait for the outcome of the transaction.
-- Show the completed transaction to the user.
-- View all the interactions for a given account with a specific account, i.e., withdraws and deposits.
+- Query historical transactions and interactions for a given account with a specific account, i.e., withdraws and deposits.
 
 ## Accounts on Aptos
 
-An [account](https://aptos.dev/concepts/basics-accounts) represents a resource on the Aptos blockchain that can send transactions. Each account is identified by a particular 32-byte account address and is a container for Move modules and Move resources. On Aptos, accounts must be created on-chain prior to any operations involving that account. The Aptos framework supports implicitly creating accounts when transferring Aptos coin via `account::transfer`.
+An [account](https://aptos.dev/concepts/basics-accounts) represents a resource on the Aptos blockchain that can send transactions. Each account is identified by a particular 32-byte account address and is a container for Move modules and Move resources. On Aptos, accounts must be created on-chain prior to any blockchain operations involving that account. The Aptos framework supports implicitly creating accounts when transferring Aptos coin via `account::transfer` or explicitly via `account::create_account`.
+
+At creation, an [Aptos account]() contains:
+* A [resource containing Aptos Coin]() and deposit and withdrawal of coins from that resource.
+* An authentication key associated with their current public, private key(s).
```

Comment:
Curious, are the empty `()` meant to include linked URLs later?

Reply:
oops yes distracted